src/traces/pointcloud/defaults.js

```diff
@@ -0,0 +1,41 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var Lib = require('../../lib');
+
+var attributes = require('./attributes');
+
+module.exports = function supplyDefaults(traceIn, traceOut, defaultColor) {
+    function coerce(attr, dflt) {
+        return Lib.coerce(traceIn, traceOut, attributes, attr, dflt);
+    }
+
+    coerce('x');
+    coerce('y');
+    coerce('bounds');
+
+    if(traceIn.xy && traceIn.xy instanceof Float32Array) {
```

Comment:
We'll abstract this logic out at some point on our way to https://github.com/plotly/plotly.js/issues/860

But, this is great for now :+1: